content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -13,7 +13,7 @@ is_beta: true
 First, configure the Datadog Agent. Our recommended method of installation for Kubernetes environments is via Helm Charts. Learn how to configure the [Datadog Agent in Kubernetes via Helm Charts][3] by reading our documentation.
 
 ## Unified Service Tagging
-For your Kubernetes applications that send traces to Datadog Agent (version 7.39 and higher), you can configure the Datadog admission controller to inject  Java and JavaScript libraries automatically.
+For your Kubernetes applications that send traces to Datadog Agent, you can configure the Datadog admission controller to inject Java, JavaScript and Python libraries automatically. To inject the Java and Javascript libraries, download Datadog Agent v7.39+. To inject the Python library, download Datadog Agent v7.40+.
```

Comment:
```suggestion
For your Kubernetes applications that send traces to Datadog Agent, you can configure the Datadog admission controller to inject Java, JavaScript and Python libraries automatically. To inject the Java and Javascript libraries, use Datadog Cluster Agent v7.39+. To inject the Python library, use Datadog Cluster Agent v7.40+.
```

Reply:
Nit as the customer don't download the agent manually, they set the agent version in their yaml/values or just use the default one in the helm chart.

Reply:
+1

Reply:
also add a comma "... to inject Java, JavaScript, and Python..."